libmogwai-tariff/tests/tariff.c

```diff
@@ -25,8 +25,12 @@
 #include <glib.h>
 #include <gio/gio.h>
 #include <libmogwai-tariff/tariff.h>
+#include <libmogwai-tariff/tariff-builder.h>
+#include <libmogwai-tariff/tariff-loader.h>
 #include <locale.h>
 
+#include "common.h"
+
```

Comment:
Is this extra newline intended?

Reply:
Yup, for better or worse it’s part of my coding style.